.github/workflows/dependency-check.yml

```
@@ -0,0 +1,40 @@
+name: OWASP Maven Dependency Check
+on:
+  schedule:
+    - cron: '0 7 * * 0'
+  workflow_dispatch:
+
+
+jobs:
+  check-dependencies:
+    name: Check dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          show-progress: false
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+          cache: 'maven'
+      - name: Run org.owasp:dependency-check plugin
+        id: plugin-run
```

choose a more descriptive name, e.g.

```suggestion
        id: dependency-check
```

---

Changed in https://github.com/cryptomator/cryptofs/pull/192/commits/60b98e43a14a8b3d7e2bb835307cbef05c4ecf4b